src/components/Contact/ContactLinks.jsx

```diff
@@ -16,7 +16,7 @@ const LinksContainer = styled.div`
     `}
 `;
 
-const Link = styled.a`
+export const Link = styled.a`
```

Comment:
Don't export components from here for use in the footer. If you see that this component can be deduplicated, then extract it to `ContentStyles.js` and import it into these two components.